playbooks/roles/tarsnap/tasks/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 
-- include: tarsnap.yml tags=tarsnap
+- include: install.yml tags=install_tarsnap
```

Comment:
I would prefer, if possible, to keep the tags be the same as the role names. That is the style we've been using elsewhere throughout the repo, e.g. the `filebeat` role will receive a `filebeat` tag so that we can just pass `--tag filebeat` if we want to run only that role. `install_` feels redundant unless there is a conflict somewhere giving undesirable behavior, and there really shouldn't be now that we're merging things.

Reply:
Nevermind, I see that this is just for a specific task file.